lib/pages/home.dart

```diff
@@ -1,34 +1,132 @@
 import 'package:flutter/material.dart';
-import 'package:flutter_map/flutter_map.dart';
-import 'package:latlong2/latlong.dart';
+import 'package:quickalert/quickalert.dart';
+import 'package:regatta_buddy/pages/regatta_details.dart';
+import 'package:regatta_buddy/pages/route_creator.dart';
+import 'package:regatta_buddy/pages/search.dart';
+import 'package:regatta_buddy/pages/user_regattas.dart';
 import 'package:regatta_buddy/widgets/app_header.dart';
 
-class HomePage extends StatefulWidget {
-  const HomePage({super.key});
-
-  static const String route = '/';
-
-  @override
-  State<HomePage> createState() => _HomePageState();
-}
-
 class _HomePageState extends State<HomePage> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: const AppHeader(),
-      body: FlutterMap(
-        options: MapOptions(
-          center: const LatLng(54.372158, 18.638306),
-          zoom: 9.2,
-        ),
-        children: [
-          TileLayer(
-            urlTemplate: 'https://tile.openstreetmap.org/{z}/{x}/{y}.png',
-            userAgentPackageName: 'com.regattaBuddy.app',
+        appBar: const AppHeader(),
+        body: Container(
+          color: Colors.green,
+          constraints: const BoxConstraints.expand(),
```

Comment:
I don't say it is better approach, but does Flexible wrapper behave the same way?

Reply:
I think it behaves the same way but I would say that this is more readable and understandable